style.css

```diff
@@ -1,108 +1,131 @@
 /*Style sheet*/
 
```

Comment:
Font was switched, CSS variables are implemented to allow for theme switching and made changes to images on index page to fix bug where they don't stack properly when in mobile view.